Sample/MediaStorageSample.xcodeproj/xcuserdata/tomoko.xcuserdatad/xcschemes/MediaStorageSample.xcscheme

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this file should not be committed because it's contained in `.gitignore` settings.